src/utils/eis_utils/src/eis_coms.c

```diff
@@ -32,17 +32,17 @@
 /**
  * @brief Unix Domain Socket (UDS) for the Identity Service API
  */
-#define EIS_UDS_IDENTITY_SOCKET_PATH "/run/aziot/identityd.sock"
+#define EIS_UDS_IDENTITY_SOCKET_PATH "/var/sockets/aziot/identityd.sock"
```

Comment:
Make this a compile time value. We need to continue to support non-snap version.